playground/lib/__tests__/lib.spec.ts

```diff
@@ -82,6 +82,17 @@ describe.runIf(isBuild)('build', () => {
     expect(umd).toMatch('process.env.NODE_ENV')
   })
 
+  test('terser minification for es lib mode', () => {
+    const terserEs = readFile('dist/terser/my-lib-custom-filename.js')
+    expect(terserEs).not.toMatch('console.log')
+  })
+
+  test('pure annotations respected by terser for tree-shaking', () => {
+    const terserEs = readFile('dist/terser/my-lib-custom-filename.js')
+    // Verify that the createObject function is present (not tree-shaken) since it's used
+    expect(terserEs).toMatch(/Date\.now/)
```

Comment:
```suggestion
  test('pure annotations are not removed by terser for es', () => {
    const terserEs = readFile('dist/terser/my-lib-custom-filename.js')
    // Verify that the createObject function is present (not tree-shaken) since it's used
    expect(terserEs).toMatch(/Date\.now/)
```

Reply:
please check that the pure annotation comment is preserved instead of checking `Date.now`